ci/templates/configure-cloud-foundry-steps.yaml

```diff
@@ -2,7 +2,7 @@ steps:
 - bash: |
     cat >> user.ini << EOF
     [behave.userdata]
-    cleanup = no
+    cleanup = yes
```

Comment:
According to https://cli.cloudfoundry.org/en-US/v6/delete-space.html, this should be all we do (note the force switch):
> cf delete-space SPACE -o ORG -f

I suspect the `cleanup` variable isn't what we need. It is documented in `behave.ini` as:
> whether to teardown artifacts after the completion of a scenario

Instead, we'd like to destroy the entire space at the end of running all tests.